libtransmission/peer-msgs.cc

```diff
@@ -164,7 +164,11 @@ auto constexpr KeepaliveIntervalSecs = int{ 100 };
 
 auto constexpr MetadataReqQ = int{ 64 };
 
-auto constexpr ReqQ = int{ 512 };
+// libtorrent-rasterbar defaults to 2000
+auto constexpr ReqQ = int{ 2000 };
+
+// libtorrent-rasterbar defaults to 500
```

Comment:
Same issues as with the comment above; but also, there appears to be no other documentation about what `ReqC` is for so there should be a couple of lines here to explain what it is.